drivers/hmis/app/graphql/types/hmis_schema/enums/ce_referral_status.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Types
+  class HmisSchema::Enums::CeReferralStatus < Types::BaseEnum
+    value 'initialized'
+    value 'in_progress'
+    value 'accepted'
+    value 'rejected'
```

Comment:
Maybe generate this enum based on  `Hmis::Ce::Referral.aasm.states.map(&:name)` so its not defined twice